src/streams/HashedBlockStream.cpp

```diff
@@ -256,3 +256,7 @@ bool HashedBlockStream::writeHashedBlock()
 
     return true;
 }
+
+bool HashedBlockStream::atEnd() const {
```

Comment:
Rename to isComplete() or isFinished() something like that.

Reply:
This overrides `QIODevice::atEnd`, which *should have been done from the start*

Reply:
Ah got it, disregard than